### PR TITLE
Modification of the documentation: details and examples added for the Tracing / Resource Usage section

### PR DESCRIPTION
**Pull request by phupe**
Following the discussions and modifications related to Issues #986 and #980, I have modified the documentation accordingly.

I have also added many examples and details in order to help the users to understand and efficiently use the very powerful report generated by nextflow.

I had to modify the `conf.py` file for sphinx in order to add the mathjax support. It might be required to install a package such as `libjs-mathjax` for the documentation to compile.

I hope you will be able to integrate these modifications into the documentation.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '19.01'
+version = '19.02'
 # The full version, including alpha/beta/rc tags.
-release = '19.01.0-edge'
+release = '19.02.0-SNAPSHOT'
```
Comment (pditommaso) [inline docs/conf.py]: Please do not commit automatically created changes

**Comment by pditommaso** [comment]
Thanks a lot for submitting this documentation improvements. However I think it's too detailed at least for the inclusion in that page. I would suggest to add a separate page `metrics` providing such information and linking it from the report document with something like *Learn more about how resources are computed in the metrics documentation*. 

Also the C source should be compiled and included the in the [stress](https://github.com/nextflow-io/stress) container.

**Comment by phupe** [comment]
Thanks for the feedback. I had the same feeling that it was mpre appropriate to have a dedicated page.
I will do the suggested modifications in the doc and container.

**Comment by phupe** [comment]
Requested modifications have been made.

**Comment by pditommaso** [comment]
Excellent. Thanks for the contribution.

**Comment by pditommaso** [comment]
I've fixed some typos, please give a look.

**Comment by phupe** [comment]
Perfect!

**Pull request merged**